src/consent.ts

```diff
@@ -170,12 +295,12 @@ export default function Consent(mpInstance) {
             return jsonObject;
         },
 
-        fromMinifiedJsonObject: function(json) {
-            var state = self.createConsentState();
+        fromMinifiedJsonObject(json: IMinifiedConsentJSONObject): ConsentState {
+            const state = self.createConsentState() as ConsentState;
```

Comment:
```suggestion
            const state: ConsentState = self.createConsentState();
```

Reply:
nit, but i think it's better practice to put the type beforehand unless it is required at the end like in line 240.